changelogs/unreleased/4796-gautierdelorme-minor.md

```diff
@@ -0,0 +1,5 @@
+## Optional Client Certificate Validation
+
+By default, client certificates are required but some applications might support different authentication schemes.
```

Comment:
```suggestion
By default, when client certificate validation is configured, client certificates are required.
However, some applications might support different authentication schemes.
```

Reply:
@skriss 👍 fixed in https://github.com/projectcontour/contour/pull/4796/commits/d910de6ea00b8f2d3e3e406736b82dca7894e926

Reply:
Looks like "However, some applications might support different authentication schemes." got dropped, was that intended? I'd keep that sentence in for clarity.

Reply:
Good catch, it wasn't intended. Fixed in https://github.com/projectcontour/contour/pull/4796/commits/45a2250fc53d087bb4b053479492e88df0fa9a6c.